r/R/csv.R

```diff
@@ -456,24 +458,31 @@ CsvReadOptions$create <- function(use_threads = option_use_threads(),
 }
 
 readr_to_csv_write_options <- function(include_header,
-                                       batch_size = 1024L) {
+                                       batch_size = 1024L,
+                                       na = "NA") {
   assert_that(is_integerish(batch_size, n = 1, finite = TRUE), batch_size > 0)
   assert_that(is.logical(include_header))
+  assert_that(is.character(na))
+  assert_that(length(na) == 1)
+  assert_that(!grepl('"', na), msg = "na argument must not contain quote characters.")
```

Comment:
Only `"` is invalid? What about `'`?

Reply:
Just `"` 😇 

https://github.com/apache/arrow/blob/b9dd41607cb7dd7afd50e3ceb99c68e79e7733a0/cpp/src/arrow/csv/writer.cc#L104